Cargo.toml

```diff
@@ -75,5 +75,5 @@ test-case = "2.2.2"
 testutils = { path = "lib/testutils" }
 
 [features]
-default = ["vendored-openssl"]
+default = ["vendored-openssl", "jujutsu-lib/legacy-thrift"]
 vendored-openssl = ["git2/vendored-openssl", "jujutsu-lib/vendored-openssl"]
```

Comment:
Perhaps, we'll need `jujutsu-lib = { .. default-features = false }` to enable `legacy-thrift` conditionally.

Reply:
Good catch. It seems to work correctly after making that change. I tested by `cargo clean; cargo build --no-default-features | grep thrift`. There's got to be a better way to test :)